src/definitions/bluebps.json

```diff
@@ -0,0 +1,254 @@
+                "payload": [
+                    {
+                        "name": "battery_voltage",
+                        "type": "u16",
+                        "description": "The main battery voltage",
+                        "units": "millivolt"
+                    },
+                    {
+                        "name": "battery_current",
+                        "type": "u16",
+                        "description": "The current measurement",
+                        "units": "centiAmp"
+                    },
+                    {
+                        "name": "battery_temperature",
+                        "type": "u16",
+                        "description": "The battery temperature",
+                        "units": "decidegrees Centigrade"
+                    },
+                    {
+                        "name": "cpu_temperature",
+                        "type": "u16",
+                        "description": "The cpu temperature",
+                        "units": "millisecond"
+                    },
+                    {
+                        "name": "flags",
+                        "type": "u8",
+                        "description": "flags indicating if any of the configured limits are currently exceeded"
+                    },
+                    {
+                        "name": "cell_voltages",
+                        "type": "vector",
+                        "vector": {
+                            "sizetype": "u8",
+                            "datatype": "u16",
+                            "size": "dynamic"
+                        },
+                        "description": "Array containing cell voltages",
+                        "units": "millisecond"
+                    }
+                ]
+            },
+            "events": {
+                "id": "9107",
+                "description": "A record of events causing a power lock-out. These numbers are non-volatile and reset only with the erase_flash control message.",
+                "payload": [
+                    {
+                        "name": "voltage",
+                        "type": "u16",
+                        "description": "The number of under-voltage events"
+                    },
+                    {
+                        "name": "current",
+                        "type": "u16",
+                        "description": "The number of over-current events"
+                    },
+                    {
+                        "name": "temperature",
+                        "type": "u16",
+                        "description": "The number of over-temperature events"
+                    }
+                ]
+            }
+        },
+        "control": {
+            "erase_flash": {
+                "id": "1100",
+                "description": "Erase flash, including parameter configuration and event counters. The mcu has a limited number of write/erase cycles (1k)!",
```

Comment:
Is it possible to know how many times a device was flashed ? Is there a register or maybe reserving part of the memory to hold this value ?

Reply:
I think I don't have it, but it is possible to add it.

Reply:
What are you thinking for this? To not exceed the erase limit specification (1000x)?

Reply:
To identify problem is the program or the user is doing something wrong with the sensor.